07-routing-gossip.md

```diff
@@ -276,6 +276,10 @@ in the `flags` field
 to indicate which end this is.  It can do this multiple times, if
 it wants to change fees.
 
+A node MAY still create a `channel_update` to communicate the channel parameters to the other endpoint, even though the channel has not been announced, e.g., because the `channels_public` bit was not set.
```

Comment:
Well, it's only in that direction, isn't it? If the node is a deadend it can also just not send the `channel_update` since the other side cannot possibly route a payment through it :wink: